readme.md

```diff
@@ -42,7 +44,8 @@ theBody.style.backgroundColor = "red";
 ```
 
 > Your answer...
-
+querySelectorAll returns all of the matching elements, since there is only one body you need to specify that you only want the first one
+theBody[0].style.backgroundColor = "red";
```

Comment:
:+1:
I prefer to avoid selecting a particular item from an array. Please see the answer key for other options.